### ISSUE TITLE
Add Aileron and Vegur

### ISSUE THREAD

**Issue by adrientetar** [google/fonts#13]
http://dotcolon.net/font/aileron/
&
http://dotcolon.net/font/vegur/

**Comment by davelab6** [comment]
https://github.com/ssagano/Aileron/pull/1

**Comment by adrientetar** [comment]
@davelab6 He has already licensed his fonts to Public Domain. Is that unacceptable for GOOG?

**Comment by davelab6** [comment]
Yes. His website says CC-0, which is as close to PD as you can get, so I
believe anyone could fork it to be OFL, but I hope that won't be necessary
and he will be willing to license it under both.

**Comment by adrientetar** [comment]
Why though?

**Comment by adrientetar** [comment]
Creating a "publicdomain" folder on the repo seems feasible to me.

**Comment by davelab6** [comment]
> Creating a "publicdomain" folder on the repo seems feasible to me.

But there is additional legal and engineering work that this implies, which won't happen.

**Comment by adrientetar** [comment]
> But there is additional legal and engineering work that this implies, which won't happen.

What do you mean?

**Comment by davelab6** [comment]
The legal department would need to approve it; then the various fonts APIs would need to be engineered to support it; then the www.google.com/fonts directory would need to be engineered to support it; then support work would be needed to maintain it.

**Comment by adrientetar** [comment]
Okay. Then again, I don't think any work is needed from the designer: you can just take the font files and change license to Apache, no?

**Comment by adrientetar** [comment]
New font in the collection, [Route159](https://github.com/ssagano/Route159).

**Comment by BafS** [comment]
Any new about Aileron ? I would love to use this font with google fonts.

**Comment by StefanPeev** [comment]
Meanwhile I forked Aileron and made my proposal for Cyrillic Script in it: https://github.com/StefanPeev/Aileron/pull/1. I'll continue to prepare the Cyrillic Script for the hole family but will be glad if you keep me informed about any news for Aileron.

**Comment by StefanPeev** [comment]
@davelab6 I forked also [Route159](https://github.com/StefanPeev/Route159) with a proposal for Cyrillic Script – see my [PR](https://github.com/ssagano/Route159/pull/1).

**Comment by converge** [comment]
any news about Aileron font ?

**Comment by fanninpm** [comment]
For those of you tired of waiting for it to show up on Google Fonts, it's available on [brick.im](https://brick.im/).

**Comment by chrissimpkins** [comment]
We are not able to identify sources for the Aileron project

**Comment by BafS** [comment]
@ssagano you are the author of those fonts, maybe you could help?

**Comment by mon-jai** [comment]
@ssagano deleted source repo for the two font 1-2 years ago.

Below are forks of the original repo before the deletion.

- https://github.com/font-archive/Aileron-source
- https://github.com/font-archive/Vegur-source
